### PR TITLE
fix index install timings

### PR DESCRIPTION
**Pull request by mohit-sheth**
eliminate ES_SERVER and use regex to trim down already supplied ELASTIC_CURL_URL by the user
added port 80 to default so that backpack can use it is -p 80

### CODE REVIEW

**Review by jtaleric — APPROVED** [review]
LGTM

**Comment by jtaleric** [comment]
Eventually, I would like to move to a more simple approach to capture the ES_Server that can be used across curl/python scripts.

File: OCP-4.X/roles/post-install/files/index-install-data.sh [hunk 1]
```diff
@@ -16,8 +16,10 @@ timestamp=`date +"%Y-%m-%dT%T.%3N"`
 rm -rf metadata-collector
 git clone http://github.com/cloud-bulldozer/metadata-collector/
 cd metadata-collector
-url=$(echo $ELASTIC_SERVER | cut -d ":" -f 1)
-port=$(echo $ELASTIC_SERVER | cut -d ":" -f 2)
+proto="$(echo $ELASTIC_URL | grep :// | sed -e's,^\(.*://\).*,\1,g')"
+ELASTIC_URL="$(echo ${ELASTIC_URL/$proto/})"
+url=$(echo $ELASTIC_URL | cut -d ":" -f 1)
```
Comment (chaitanyaenr) [inline OCP-4.X/roles/post-install/files/index-install-data.sh:21]: s/url/server

**Review by chaitanyaenr — COMMENTED** [review]
Minor nit

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**